### PR TITLE
adding web driver

### PR DESCRIPTION
**Pull request by Paola-Xaq**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,7 +1,11 @@
 plugins {
     id 'java'
+    id 'checkstyle'
 }
 
+//sourceCompatibility = 12
```
Comment (JorgeRCV) [inline build.gradle]: This lines could be deleted

**Review by JorgeRCV — APPROVED** [review]

File: src/main/java/core/utils/EnvConfiguration.java [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+import java.util.Properties;
+
+public class EnvConfiguration {
+    public static Properties CONFIG = new Properties();
+
+    static {
+        try {
+            CONFIG = read();
+        } catch (IOException e) {
+            e.printStackTrace();
```
Comment (JorgeRCV) [inline src/main/java/core/utils/EnvConfiguration.java]: It should manage the exception, not only print the error.

File: src/main/java/core/utils/EnvConfiguration.java [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+        Properties properties = new Properties();
+        File file = new File(CONFIG_PATH);
+
+        if (!file.exists()) {
+            file.createNewFile();
+        }
+        try (InputStream reading = new FileInputStream(file)) {
+            properties.load(reading);
+        } catch (Exception e) {
+            e.printStackTrace();
```
Comment (JorgeRCV) [inline src/main/java/core/utils/EnvConfiguration.java]: It should manage the exception, not only print the error.

File: src/main/java/salesforce/pages/BasePage.java [hunk 4]
```diff
@@ -0,0 +1,27 @@
+package salesforce.pages;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public abstract class BasePage {
+    protected WebDriver driver;
+    protected WebDriverWait wait;
+    private static final int WAIT_TIME = 30;
```
Comment (JorgeRCV) [inline src/main/java/salesforce/pages/BasePage.java:10]: You could provide this value on the environment file.

File: src/main/java/core/utils/EnvConfiguration.java (hunk 2 above)
Comment (Paola-Xaq) [reply src/main/java/core/utils/EnvConfiguration.java]: Exception is handled in the Properties read() method.

**Pull request merged**